src/javascript/app_2/Stores/Modules/Trading/Helpers/contract_type.js

```diff
@@ -159,6 +161,10 @@ const ContractType = (() => {
         duration_units_list: getPropertyValue(available_contract_types, [contract_type, 'config', 'durations', 'units_display', contract_start_type]) || [],
     });
 
+    const getDurationUnitsMinMax = (contract_type, contract_start_type) => ({
```

Comment:
is this different from the function on line 181, `getDurationMinMax`?

Reply:
@negarn 
There is a difference between them, the one on line 164 doesn't need `contract_start_type` but in the one on line 181 needs it. it's possible to merge both functionalities in one function.

Reply:
would be good to merge them then, since the other is unused